### DISCUSSION TITLE
Add padding between devicon and file?

### DISCUSSION THREAD

**Discussion by HoldenLucas** [nvim-telescope/telescope.nvim#1689] [Q&A]
Is it possible to add a space between devicons and the file names? I'm doing something similar to [this](https://github.com/kyazdani42/nvim-web-devicons/issues/102#issuecomment-998255906) to add a space after every icon, but I'd like to avoid redefining the icon itself.

**Comment by Conni2461** [comment]
there is a space after the icon https://github.com/nvim-telescope/telescope.nvim/blob/e8ccd47c4f748a0db5c84c84e2bc7f518cb94cea/lua/telescope/utils.lua#L455

About which picker are we talking
